core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1508,21 +1508,21 @@ private static RelDataType arrayCompactReturnType(SqlOperatorBinding opBinding)
           OperandTypes.AT_LEAST_ONE_SAME_VARIADIC);
 
   /** The "ARRAY_CONTAINS(array, element)" function. */
-  @LibraryOperator(libraries = {SPARK})
+  @LibraryOperator(libraries = {SPARK, HIVE})
```

Comment:
``` 
select array_contains(array(1,null),2);
```
In Spark, returns NULL
But In Hive, it returns FALSE
This cannot be the same function.

Reply:
Have you checked the results of executing different parameters or values in Hive?

Reply:
Thanks for your reminder, had change array_contains with Spark, other array functions I would check it again. @NobiGo

Reply:
> ```
> select array_contains(array(1,null),2);
> ```
> 
> In Spark, returns NULL But In Hive, it returns FALSE This cannot be the same function.

According discussion with Spark community，this is a bug of spark，and here array_contains(array(1,null),2) should return false not null, at the same time test array_contains(array(1, null),1) would return true in Spark，can we not consider about it currently？other functions test is well with different parameters.  @NobiGo

Reply:
Thanks for checking @xuzifu666 , has a ticket been created for that bug in Spark?

Reply:
Yes, I had submit a issue in Spark JIRA: https://issues.apache.org/jira/browse/SPARK-51319 @rubenada